inference-engine/src/vpu/common/include/vpu/utils/simple_math.hpp

```diff
@@ -26,7 +26,7 @@ namespace vpu {
 
 template <typename T>
 Optional<int> parseNumber(const std::string& s) {
-    T value;
+    T value{};
```

Comment:
What an issue was here?

Reply:
> 'value' is used uninitialized in this function.

Reply:
Here, I would agree, since `T` could be an int